breathecode/mentorship/views.py

```diff
@@ -689,22 +690,32 @@ def get(self, request, bill_id=None, academy_id=None):
 
     @capable_of('read_mentorship_bill')
     def put(self, request, bill_id=None, academy_id=None):
+        many = isinstance(request.data, list)
+        if many:
+            bill = []
+            for obj in request.data:
+                bill.append(MentorshipBill.objects.filter(id=obj['id']).first())
 
-        if bill_id is None:
-            raise ValidationException('Missing bill ID on the URL', 404)
+        else:
+            if bill_id is None:
+                raise ValidationException('Missing bill ID on the URL', 404)
 
-        bill = MentorshipBill.objects.filter(id=bill_id, academy__id=academy_id).first()
-        if bill is None:
-            raise ValidationException('This bill does not exist for this academy', 404)
+            bill = MentorshipBill.objects.filter(id=bill_id, academy__id=academy_id).first()
+            if bill is None:
+                raise ValidationException('This bill does not exist for this academy', 404)
 
         serializer = MentorshipBillPUTSerializer(bill,
                                                  data=request.data,
+                                                 many=many,
                                                  context={
                                                      'request': request,
                                                      'academy_id': academy_id
                                                  })
         if serializer.is_valid():
             mentor = serializer.save()
+            if (many):
+                return Response(serializer.data, status=status.HTTP_200_OK)
+
```

Comment:
_serializer = GETBillSmallSerializer(bill, many=many)

check if `mentor = serializer.save()` return a queryset in the case of bulk request